Dockerfile

```diff
@@ -35,4 +37,10 @@ COPY --from=builder /app /app/
 
 WORKDIR /app
 
+RUN groupadd -g 1001 app && \
+    useradd app -u 1001 -g 1001 --home /app && \
+    ln -fs /tmp /app
```

Comment:
Better to create the symlink before copying the rest of the app tree over, so that there's no ambiguity about it potentially putting the symlink inside a preexisting /app/tmp that might have been copied over. e.g. see https://github.com/alphagov/smart-answers/blob/7379189/Dockerfile#L64

Reply:
i did consider having soft link creation prior the copy but as search-api does not have a tmp folder within its repo i thought it would be fine. 
Saying that things may change in the future so i will bring it above the copy. thanks

Reply:
updated and tested